games/achievements.py

```diff
@@ -188,6 +189,36 @@ def get_level(user):
         return AchievementLevel.NO_LEVEL
 
 
+class TheMoreTheMerrierAchievement(Achievement):
+    name = "The More The Merrier"
+    description = "Play atleast 5/10/15/20 games with equally as many unique players"
+    icon = "merrier.svg"
+
+    def get_level(user):
+        played_with_count = Counter()
+        for game in user.games.filter():
+            for player in game.players.all():
+                if player != user:
+                    played_with_count[player.username] += 1
+
+        top20 = sorted(
+            ({"x": k, "y": v} for k, v in played_with_count.items()),
+            key=lambda x: -x["y"],
+        )[:30]
+        if len(top20) < 20:
```

Comment:
This logic is wrong. If a player has only ever played with the same 5 players, they should be able to get the base level.